src/Tools/Microsoft.dotnet-openapi/test/OpenApiAddURLTests.cs

```diff
@@ -177,6 +177,7 @@ public async Task OpenApi_Add_Url()
     }
 
     [Fact]
+    [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/61348", Queues = "Windows.11.Amd64.Client.Open")]
```

Comment:
I think we want to quarantine, not skip these, even if it applies to all legs since we want to be able to monitor if they eventually recover from flakiness.

Reply:
And I thought that I'm doing better by skipping only on one queue. But I see your point, fair enough.